upgrade/v1beta1/upgrade.go

```diff
@@ -4,30 +4,30 @@ import (
 	"context"
 	"fmt"
 	"reflect"
-	"strings"
 
 	"github.com/jinzhu/copier"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	restclient "k8s.io/client-go/rest"
 
 	longhornV1beta1 "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta1"
-	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
-
-	"github.com/longhorn/longhorn-manager/types"
-
 	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
+	"github.com/longhorn/longhorn-manager/types"
 )
 
 const (
 	upgradeLogPrefix = "upgrade from v1beta1 to v1beta2:"
 )
 
+// UpgradeCRFromV1beta1ToV1beta2 initialize the old CR spec which is with the `null` field.
+// Since start from longhorn.io/v1beta2, we introduce the CRD structural schema.
+// However, for the Kubernetes < 1.20, there is an issue that it does not allow
+// the `null` field because of the CRD validation failure.
+// Reference issue: https://github.com/kubernetes/kubernetes/issues/86811`
 func UpgradeCRFromV1beta1ToV1beta2(config *restclient.Config, namespace string, lhClient *lhclientset.Clientset) (err error) {
```

Comment:
It's better to rename this and the below functions in this file. e.g., `FixupV1beta1CRsForConversion`